static/js/utils/tile_utils.tsx

```diff
@@ -407,7 +407,9 @@ export function getStatFormat(
     const seriesWithFacet = Object.values(
       statSeriesData.data[svSpec.statVar]
     ).find((series) => !!series.facet);
-    statMetadata = statSeriesData.facets[seriesWithFacet.facet];
+    if (seriesWithFacet) {
```

Comment:
can you also add add this same check for obsWithFacet in the statPointData handling

Reply:
Done